packages/s/snapd/files/wrapper.sh

```diff
@@ -29,15 +29,18 @@ fi
 
 if [[ "${CONFINEMENT}" != "strict" ]] && [[ "${DISABLE_CONFINEMENT_WARNING:-n}" != "y" ]]
   then
-  if [[ -n "${BAMF_DESKTOP_FILE_HINT+x}" ]] && [[ -n "${GIO_LAUNCHED_DESKTOP_FILE+x}" ]]
+  # This prevents the warning from showing on KDE Plasma
+  # if [[ -n "${BAMF_DESKTOP_FILE_HINT+x}" ]] && [[ -n "${GIO_LAUNCHED_DESKTOP_FILE+x}" ]]
+
```

Comment:
I would just remove this entirely if it doesn't work.

```suggestion
```

Reply:
Removed